code/game/machinery/computer/buildandrepair.dm

```diff
@@ -560,54 +636,54 @@
 	icon = 'icons/obj/machines/HONKputer.dmi'
 	base_mineral = /obj/item/stack/sheet/mineral/bananium
 
-/obj/structure/computerframe/HONKputer/attackby(obj/item/P as obj, mob/user as mob, params)
+/obj/structure/computerframe/HONKputer/attackby(obj/item/I, mob/user, params)
```

Comment:
Same here as above

Reply:
Looks like this is getting removed by #16213, so I'll just do the other one.